code/game/objects/effects/landmarks/corpsespawner.dm

```diff
@@ -92,9 +92,9 @@
 		W.registered_name = M.real_name
 		M.equip_to_slot_or_del(W, SLOT_WEAR_ID)
 	if(xenovictim)
-		M.adjustBruteLoss(100)
-		M.adjustBruteLoss(100)
-		M.adjustBruteLoss(100)
+		//M.adjustBruteLoss(100)
```

Comment:
More commented out code

Reply:
no its there to avoid future confusion

Reply:
This doesn't explain anything though, could there not be a commented added here to explain why this code is not here instead of commented out code?